web/packages/teleport/src/Console/Console.tsx

```diff
@@ -108,7 +108,14 @@ export default function Console() {
               disableNew={disableNewTab}
               onNew={onTabNew}
             />
-            <ActionBar onLogout={onLogout} />
+            <ActionBar
+              onLogout={onLogout}
+              latency={{
+                client: activeDoc?.latency?.client ?? 0,
+                server: activeDoc?.latency?.server ?? 0,
```

Comment:
I'm wondering if we should default to `0` if there is no value.
Maybe we can allow the whole `latency` prop to be `undefined` and then in the component show something along these lines:
<img width="540" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/ccca4e7b-db10-4ea1-bbab-f34b1a45e430">